services/shotgrid_common/utils.py

```diff
@@ -219,6 +219,7 @@ def create_ay_fields_in_sg_project(
             attribute containing the type of data and the scope of the
             attribute.
     """
+    # NOTE: this is not needed?
```

Comment:
@jakubjezek001 I noticed you added this, is this needed now? the `type` should always exist on this case since `SG_PROJECT_ATTRS` is our constant?

Reply:
Yeah this was probably passed from some code migration. We definitely need it, you are right. Thanks

Reply:
we do or we do NOT?